charts/spire/charts/spire-server/README.md

```diff
@@ -28,6 +28,56 @@ A Helm chart to install the SPIRE server.
 
 * <https://github.com/spiffe/helm-charts/tree/main/charts/spire>
 
+## Tornjak
+
+Tornjak is the UI and Control Plane for SPIRE [https://github.com/spiffe/tornjak](https://github.com/spiffe/tornjak)
+
+When Tornjak is enabled, it can be reached in one of the three connection types:
+
+* http - simple HTTP connection recommended for demonstration purpose only. It should not be use for production.
+* tls - enables Transport Layer Security (TLS) with HTTPS communication. Tornjak client can validate Tornjak server using TLS Certificate (see instructions below)
+* mtls - mutual TLS. Both, Tornjak client and the server can validate each other using provided certificates (see instructions below)
+
+### Tornjak with TLS
+
+`tornjak.config.connectionType=tls`
+
+TLS connection requires Tornjak to have access to TLS key and certificate.
+Complete instruction on creating your own TLS certificate can be found [here](https://github.com/spiffe/tornjak/blob/main/examples/tls_mtls/README.md).
+TLS Certificate and the private key must be provided to Tornjak via *TLS Secret*. Prior to deploying this Helm chart, create TLS Secret in the deployment namespace (e.g. `spire-server`)
```

Comment:
Shouldn't we just bake this in, similar like the service accounts are implemented?